seed/analysis_pipelines/better/pipeline.py

```diff
@@ -311,21 +311,48 @@ def _process_results(self, analysis_id):
     ]
 
     # gather all columns to store
+    BETTER_VALID_MODEL_E_COL = 'better_valid_model_electricity'
+    BETTER_VALID_MODEL_F_COL = 'better_valid_model_fuel'
     column_data_paths = [
         ExtraDataColumnPath(
-            'better_cost_savings_combined',
-            'BETTER Potential Cost Savings (USD)',
-            'assessment.assessment_energy_use.cost_savings_combined'
```

Comment:
@nllong these changes removes the combined cost, ghg, energy usage columns from the inventory list -- is that ok? we're now pulling out electricity/nat-gas specific values.